packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2904,6 +2923,15 @@ class InputDecoration {
   /// If [InputDecorationTheme.hintFadeDuration] is null defaults to 20ms.
   final Duration? hintFadeDuration;
 
+  /// Whether the input field's height should always be greater than or equal to
```

Comment:
With the docs the field name should probably be negated (e.g. `maintainHintHeight`).